tests/components/broadlink/test_heartbeat.py

```diff
@@ -0,0 +1,100 @@
+"""Tests for Broadlink heartbeats."""
+from unittest.mock import call, patch
+
+from homeassistant.components.broadlink.heartbeat import BroadlinkHeartbeat
+from homeassistant.util import dt
+
+from . import get_device
+
+from tests.common import async_fire_time_changed
+
+DEVICE_PING = "homeassistant.components.broadlink.heartbeat.blk.ping"
+
+
+async def test_heartbeat_trigger_startup(hass):
+    """Test that the heartbeat is initialized with the first config entry."""
+    device = get_device("Office")
+
+    with patch(DEVICE_PING) as mock_ping:
+        await device.setup_entry(hass)
+        await hass.async_block_till_done()
+
+    assert mock_ping.call_count == 1
+    assert mock_ping.call_args == call(device.host)
+
+
+async def test_heartbeat_ignore_oserror(hass):
```

Comment:
```suggestion
async def test_heartbeat_ignore_oserror(hass, caplog):
```
Please verify using caplog that a log message is written, otherwise the test might succeed without the exception ever being raised.

Reply:
Ok.